src/Particle/ParticleIO/ParticleLayoutIO.cpp

```diff
@@ -110,8 +111,10 @@ bool LatticeParser::put(xmlNodePtr cur)
       }
       else if (aname == "LR_handler")
       {
+        std::string handler_type;
+        //This chops whitespace so the simple str == comparisons work
         putContent(handler_type, cur);
-        tolower(handler_type);
+        handler_type = lowerCase(handler_type);
```

Comment:
There was a default value.

Reply:
I got rid of the other definition that was shadowed.  I copied the default initialization but it took me a while to decide the istream corner case that doesn't clear the string iff isn't UB. that aspect of putContent didn't actually catch my attention before.